cwd/app.js

```diff
@@ -3,7 +3,9 @@
 
   const allGlyphs = activeGlyphs.arr;
   views = allGlyphs.filter(obj => obj.view).map(obj => obj.view);
+  console.log(views)
```

Comment:
We shouldn't leave this `console.log` in.

Reply:
Yeah, I noticed that during office hours today, and removed it, just haven't pushed yet since other fixes are ongoing.